content/en/tests/guides/add_custom_measures.md

```diff
@@ -20,13 +20,11 @@ further_reading:
 
 ## Overview
 
-Before you begin, make sure that [Test Visibility][1] is already set up for your language. This guide walks you through adding and using custom metrics for your tests.
+Before you begin, make sure that [Test Visibility][1] is already set up for your language. This guide walks you through adding and using custom measures for your tests.
 
-<div class="alert alert-warning">These custom metrics are <strong>not</strong> <a href="/metrics">Datadog metrics</a>. They are numerical tags (also known as measures) that represent things like memory usage or request rates.</div>
```

Comment:
I don't think this is needed any longer, as we don't use the word "metrics" anymore. If anyone disagrees, let me know